src/TTextEdit.cpp

```diff
@@ -1655,7 +1655,7 @@ int TTextEdit::bufferScrollUp( int lines )
 
 int TTextEdit::bufferScrollDown( int lines )
 {
-    if( ( mpBuffer->mCursorY + lines ) < (int)(mpBuffer->size()-1 - mScreenHeight) )
+    if( ( mpBuffer->mCursorY + lines ) < (int)(mpBuffer->size()) )
```

Comment:
Background information: http://stackoverflow.com/questions/103512/in-c-why-use-static-castintx-instead-of-intx

Where would be a good place to record this decision? I'd prefer somewhere in a source file. Maybe enable errors on old C style declarations?

This PR doesn't introduce the (int) though so I think it doesn't have to fix it in order to get accepted.